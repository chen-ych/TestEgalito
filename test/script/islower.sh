#!/bin/bash
mkdir -p tmp

ln -sf ../../src/libegalito.so
LD_LIBRARY_PATH=../../src EGALITO_DEBUG=/dev/null ../../src/loader \
	../binary/build/islower 2>&1 \
    | tail -n 2 >tmp/islower.out
rm libegalito.so

if [ -z "$(diff islower.expected tmp/islower.out)" ]; then
    echo "test passed"
else
    echo "test failed! this is usually because callInitFunctions isn't running"
    exit 1
fi
