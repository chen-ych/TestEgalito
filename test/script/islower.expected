'a' should be lowercase: 512
'A' should not be lowercase: 0
